brewtils/schemas.py

```diff
@@ -161,12 +142,15 @@ def get_attribute_names(cls):
             if isinstance(value, fields.FieldABC)
         ]
 
+    class Meta:
+        unknown = EXCLUDE
+
 
 class ChoicesSchema(BaseSchema):
     type = fields.Str(allow_none=True)
     display = fields.Str(allow_none=True)
     value = fields.Raw(allow_none=True, many=True)
-    strict = fields.Bool(allow_none=True, default=False)
+    strict = fields.Bool(allow_none=True, dump_default=False)
```

Comment:
Using dump_default is ugly, but confirmed that this the way to do it in marshmallow 3.13+.